version-stable-r-development.Rmd

```diff
@@ -18,63 +18,71 @@ environment based on containerized solutions leveraging the
 of multiple dockerized development flavors, to match various target production
 environments or projects.
 
+> _The instructions in this chapter are for R >= 4.0.0.
+Images for R <= 3.6.3 are defined in 
+[rocker-org/rocker-versioned](https://github.com/rocker-org/rocker-versioned),
+but are no longer actively maintained._
```

Comment:
Added as suggested. It looks like this:

![image](https://github.com/miraisolutions/techguides/assets/19765083/f43a9c6e-2538-4815-9a68-48a7e1d1d918)